packages/fxa-js-client/tests/mocks/request.js

```diff
@@ -223,6 +227,25 @@ define(['client/lib/errors', 'tests/lib/push-constants'], function(
       status: 200,
       body: '{"subscriptions":[{"foo":"bar"}]}',
     },
+    securityEvents: {
+      status: 200,
+      body: JSON.stringify([
+        {
+          name: 'account.create',
+          verified: true,
+          createdAt: new Date().getTime(),
+        },
+        {
+          name: 'account.login',
+          verified: true,
+          createdAt: new Date().getTime(),
+        },
+      ]),
+    },
+    securityEventsEmptyResponse: {
+      status: 200,
+      body: '[]',
```

Comment:
This returns response `{}` instead of an array, ref https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/security-events.js#L38.